src/views/RecIM/components/List/index.jsx

```diff
@@ -64,9 +64,9 @@ const MatchList = ({ matches, activityID }) => {
   return <List dense>{content}</List>;
 };
 // setTargetTeamID is used for edit Match teams
-const TeamList = ({ teams, match, invite, setInvites, setTargetTeamID }) => {
+const TeamList = ({ teams, match, series, invite, setInvites, setTargetTeamID }) => {
   const navigate = useNavigate();
-  if (!teams?.length && !match) return <Typography>No teams to show.</Typography>;
+  if (!teams?.length && !match && !series) return <Typography>No teams to show.</Typography>;
```

Comment:
I think you're getting half of it, but I'm pretty sure that this is actually an all encompassing check to make sure that the team list is properly propagated. the `&& !series` checks to make sure that we're mapping proper data. However, I'm actually questioning why we even do this check at all. I think it was originally set in place when the only way we could propagate TeamList was with teams.length. Since now there are more ways to do this, we might be able to remove this entire conditional check.

Reply:
im pretty sure this conditional is a redundant check to ensure that teamslist has something passed into it

Reply:
I'm just deleting the check, just tested. It works perfectly without it.